static/src/xml/website_sale_stock_product_availability.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates>
+    <t t-extend="website_sale_stock.product_availability">
+        <t t-jquery="#threshold_message" t-operation="append">
+            <a href="#" class="gse_change_warehouse" t-att-data-gse-wh-qty="JSON.stringify(gse_warehouses_qty_custo)">
+                Check quantity in other warehouse
+            </a>
+        </t>
+    </t>
+
+    <t t-name="gse_custo.popup_content_warehouse_select">
+        <div>
+            <div class="card">
+                <label t-foreach="qty" t-as="wh">
+                    <div class="card-body d-flex" style="border-top: 1px solid #dddddd;">
+                        <div class="flex-grow-1">
+                            <span><b t-esc="wh.name"/></span><br />
+                            <p>Only <t t-esc="wh.free_qty"/> Units left in stock.</p>
+                            <i class="fa fa-circle mr-1" style="color:red;"/>Something to show here<br/>
```

Comment:
No idea so far for this line, but on the next one the address is a good idea. (the one available on the wh, obviously)